### DISCUSSION TITLE
Papers or books

### DISCUSSION THREAD

**Discussion by paris0120** [Python-Fuzzylogic/fuzzylogic#15] [Q&A] [answered]
I'm wondering which papers/books you followed for the program. I just started learning fuzzy logic and can't find a good paper or book. Thanks a lot.

**Comment by amogorkon** [comment, marked as answer]
I've learned the basics of fuzzy logic in my CS study at university, many years ago, but I can't give you those resources.
The book I followed mostly, idea and definition-wise is **Fuzzy Logic and Control: Software and Hardware Applications, Vol. 2** by Mohammad Jamshidi, Nader Vadiee and Timothy Ross
 ( https://books.google.de/books/about/Fuzzy_Logic_and_Control.html?id=fN9SAAAAMAAJ&redir_esc=y )

Oh, and there are excellent youtube videos on the subject that also inspired me:
Lecture series by Prof S Chakraverty ( https://www.youtube.com/watch?v=oWqXwCEfY78 ) 
and examples like https://youtu.be/R4TPFpYXvS0
